apps/web/src/pages/get-started/tabs/Setup.tsx

```diff
@@ -47,7 +47,7 @@ export const buildGuides = [
               </TextElement>
 
               <CodeEditor
-                height="200px"
+                height="300px"
```

Comment:
Making sure that the full code snippet is shown immediately without a need to scroll.